From 52172502cf2dcbc5e9f7df5f74bbe1ba7c450e1e Mon Sep 17 00:00:00 2001
From: Dave Grote <grote1@llnl.gov>
Date: Thu, 5 Nov 2020 10:17:28 -0800
Subject: [PATCH] Fix includes in RhoFunctor.cpp

---
 Source/Diagnostics/ComputeDiagFunctors/RhoFunctor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/Diagnostics/ComputeDiagFunctors/RhoFunctor.cpp b/Source/Diagnostics/ComputeDiagFunctors/RhoFunctor.cpp
index e48703e917..f1dab4157f 100644
--- a/Source/Diagnostics/ComputeDiagFunctors/RhoFunctor.cpp
+++ b/Source/Diagnostics/ComputeDiagFunctors/RhoFunctor.cpp
@@ -2,7 +2,7 @@
 #include "RhoFunctor.H"
 #include "Utils/CoarsenIO.H"
 
-#ifdef WARPX_DIM_RZ
+#if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
 #include "FieldSolver/SpectralSolver/SpectralFieldData.H"
 #endif
 
